src/stim/circuit/circuit_repeat_block.pybind.h

```diff
@@ -19,8 +19,6 @@
 
 #include "stim/circuit/circuit.h"
 
-void pybind_circuit_repeat_block(pybind11::module &m);
-
 struct CircuitRepeatBlock {
```

Comment:
Ah, leftovers from the incremental "move it into stim_pybind when I touch it" changes. This should also go in the `stim_pybind` namespace.

Reply:
:+1: